src/pages/basic_data/institution_admin/FindPersonPanel.tsx

```diff
@@ -81,6 +88,26 @@ export const FindPersonPanel = () => {
                   />
                 )}
               </Field>
+
+              <Field name="user.nvi.verifiedAt.id">
+                {({ field, form }: FieldProps<NviVerification>) => (
+                  <FormControlLabel
+                    sx={{ whiteSpace: 'pre-line' }}
+                    control={
+                      <Checkbox
+                        {...field}
+                        value={values.user.nvi}
+                        onChange={() => {
```

Comment:
Må fortsatt trykke to ganger på checkbox for at den skal velges om NIN-feltet er i fokus, så det er en bug vi burde løse tenker jeg

Reply:
Funker greit hos meg :thinking:

Reply:
Ser ut som dette er nokså innfløkt, og skjer når man klikker øverst i checkboxen. Om et tekstfelt over trigger feilmelding så flyttes alt innhold under litt ned, og museklikket treffer ikke lenger checkbox. Ganske snever bug, som man kan løse ved å trykke igjen, så tenker det er greit å se mellom fingrene for akkurat denne nå.

Løsningen vil være at innholdet ikke hopper når man trigger en feilmelding